src/api/migrations/20190704140437-create-share.js

```diff
@@ -1,49 +1,34 @@
-
 export default {
-  up: (queryInterface, Sequelize) => {
-    return queryInterface.createTable('Shares', {
-      id: {
-        allowNull: false,
-        autoIncrement: true,
-        primaryKey: true,
-        type: Sequelize.INTEGER
-      },
-      userId: {
-        type: Sequelize.INTEGER,
-        references: {
-          model: 'Users',
-          key: 'id'
-        },
-        onDelete: 'CASCADE',
-        onUpdate: 'CASCADE',
-        allowNull: false,
-        unique: false
-      },
-      articleSlug: {
-        type: Sequelize.STRING,
-        references: {
-          model: 'Articles',
-          key: 'slug'
-        },
-        onDelete: 'CASCADE',
-        onUpdate: 'CASCADE',
-        allowNull: false,
-        unique: false
-      },
-      sharedOn: {
-        type: Sequelize.STRING
-      },
-      createdAt: {
-        allowNull: false,
-        type: Sequelize.DATE
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Shares', {
```

Comment:
@murediane why are you manually editing this migration file instead of creating a new one with the changes you want to make to the table? I believe we spoke about this while you were working on the backend.

Reply:
yeah, I remember, I am sorry to let me fix it.